TodoApp/TodoApp.Api/Models/Item.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace TodoApp.Api.Models
+{
+    [Serializable]
+    public class Item
+    {
+        public Item(string text, Guid id)
+        {
+            Text = text;
+            Id = id;
+        }
+
+        public Guid Id;
+        public string Text;
```

Comment:
One more thing: Have you tried to fail the tests? Like, change a digit in a `GUID` in a container? 
The golden rule is to always verify that tests fail and fail with a reasonable message - I'm not sure this is the case right now 😉

Reply:
You still didn't fix failing tests scenario. When a test fails, it prints: `Expected: <TodoApp.Api.Models.Item>
  But was:  <TodoApp.Api.Models.Item>` - this is not very helpful, is it? Especially since the string is same while sentence suggests otherwise :wink: